seed/static/seed/js/controllers/analyses_controller.js

```diff
@@ -23,7 +23,7 @@ angular.module('BE.seed.controller.analyses', [])
       analyses_service,
       Notification,
     ) {
-      $scope.org = organization_payload.organization;
+      $scope.org = organization_payload;
```

Comment:
The data passed in for organization_payload has changed -- it's no longer nested